### ISSUE TITLE
Why is the image training loss very poor?

### ISSUE THREAD

**Issue by zhangyong333** [Mondo-Robotics/DiT4DiT#9]
<img width="1445" height="720" alt="Image" src="https://github.com/user-attachments/assets/5c76c946-f035-4565-8d9f-d01654ca896d" />

<img width="1414" height="706" alt="Image" src="https://github.com/user-attachments/assets/7ce51056-2bb8-4f9e-8f49-7f1c712693e0" />

**Comment by TeleeMa** [comment]
It is normal. The degree of convergence was not particularly high because, unlike common practice, our video training did not use logit normal for the time distribution; instead, we employed uniform sampling. While training around t=0.5 would have improved generation quality and resulted in greater convergence, it would have had a detrimental effect on the action component.

You can use logit normal to get better generation quality by specify 
`--framework.cosmos25.flow_matching.time_distribution logit_normal `
in the training scripts.
